recipe/bld.bat

```diff
@@ -7,82 +7,85 @@ REM Compile python, extensions and external libraries
 if "%ARCH%"=="64" (
    set PLATFORM=x64
    set VC_PATH=x64
-   set BUILD_PATH=amd64
+   set PCB=%SRC_DIR%\PCbuild\amd64
+   call "C:\Program Files (x86)\Microsoft Visual Studio 9.0\VC\bin\vcvars64.bat"
 ) else (
    set PLATFORM=Win32
    set VC_PATH=x86
-   set BUILD_PATH=win32
+   set PCB=%SRC_DIR%\PCbuild
+   call "C:\Program Files (x86)\Microsoft Visual Studio 9.0\VC\bin\vcvars32.bat"
```

Comment:
Why is this necessary?  Shouldn't conda-build be taking care of this?  Also, by defining these, I think that you are precluding use of the Visual C++ compiler for Python tools, which is really the only way that most people have access to these compilers, especially on 64-bit.